pkg/serviced.default

```diff
@@ -1,194 +1,247 @@
+#   for the corresponding internal service.  The value of the variable
+#   is of the form SVC:KEY=VAL, where service is the name of the
+#   internal service, KEY is the environment variable to set for that
+#   service, and VAL is the value to which to set the variable.
+# SERVICED_ISVCS_ENV_0=elasticsearch-logstash:ES_JAVA_OPTS=-Xmx4g
 
 # Set the ISVCS path for serviced internal data
 # SERVICED_ISVCS_PATH=/opt/serviced/var/isvcs
 
-# Set the VOLUMES path for serviced application data
-# SERVICED_VOLUMES_PATH=/opt/serviced/var/volumes
+# Enables the following isvcs to run on the host (if not the master)
+# SERVICED_ISVCS_START=celery,elasticsearch,logstash,opentsdb,docker-registry,zookeeper,kibana
 
-# Set the BACKUPS path for serviced backups
-# SERVICED_BACKUPS_PATH=/opt/serviced/var/backups
+# Specify the id no. of the zookeeper cluster node (must be unique amongst all
+# of the nodes in the cluster)
+# SERVICED_ISVCS_ZOOKEEPER_ID=
+
+# Specify nodes in the zookeeper quorum if this host is running as part of the
+# zookeeper quorum.  This takes the form of <ZKID#>@<IPAddress>:<PeerPort>:<LeaderPort>
+# SERVICED_ISVCS_ZOOKEEPER_QUORUM=1@host1:2888:3888,2@host2:2888:3888,3@host3:2888:3888
 
 # Set the TLS keyfile
 # SERVICED_KEY_FILE=/etc/....
 
-# Set the TLS certfile
-# SERVICED_CERT_FILE=/etc/....
+# Logstash purging cycle time in hours
+# SERVICED_LOGSTASH_CYCLE_TIME=6
 
-# Set the minimum supported TLS version for HTTP connections, valid values VersionTLS10|VersionTLS11|VersionTLS12
-# SERVICED_TLS_MIN_VERSION=VersionTLS10
+# Set the address for the logstash elastic search
+# SERVICED_LOGSTASH_ES={{SERVICED_MASTER_IP}}:9100
 
-# Set the supported TLS ciphers for HTTP connections
-# SERVICED_TLS_CIPHERS=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_3DES_EDE_CBC_SHA,TLS_RSA_WITH_RC4_128_SHA,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_RC4_128_SHA,TLS_ECDHE_RSA_WITH_RC4_128_SHA
+# Set the age (in days) of logstash data to keep
+# SERVICED_LOGSTASH_MAX_DAYS=14
 
-# Set the driver type on the master for the distributed file system (rsync/btrfs/devicemapper)
-# SERVICED_FS_TYPE=devicemapper
+# Max size of Logstash data to keep in gigabytes
+# SERVICED_LOGSTASH_MAX_SIZE=10
 
-# Set to true to allow use of loopback files (instead of thin pools) with devicemapper for serviced storage.
-# NOTE: This is not recommended for production use.
-# SERVICED_ALLOW_LOOP_BACK=false
+# Set the address to which to send logs
+# SERVICED_LOG_ADDRESS={{SERVICED_MASTER_IP}}:5042
+
+# Set the loglevel for serviced
```

Comment:
On the other hand.. the logging descriptions in general are in the other ticket.  Does this apply to both glog and the package logger? Would it be better to leave this until we've updated the descriptions on logging completely?